packages/@tinacms/cli/src/next/commands/dev-command/index.ts

```diff
@@ -70,70 +71,77 @@ export class DevCommand extends BaseCommand {
     const setup = async ({ firstTime }: { firstTime: boolean }) => {
```

Comment:
Rather than pass in "firstTime", would it be cleaner to just check is database is undefined?

Reply:
Sorry this was mixed in with my branch, it's not part of the PR, will fix

Reply:
Alternatively, if part of this is only executed on the first time, maybe we should split this into 2 functions? (`init`, and something else)?